doc/developer/testing.rst

```diff
@@ -7,14 +7,15 @@ Quick instructions
 ##################
 
 To run the tests, assuming that all requirements have been installed
+
 1. Activate your environment with SmartSim and SmartRedis installed
 2. Modify `SR_DB_TYPE` (`Clustered` or `Standalone`) and
    `SMARTREDIS_TEST_DEVICE` (`gpu` or `cpu`) as necessary in
    `setup_test_env.sh`.
 3. `source setup_test_env.sh`
 4. `pushd utils/create_cluster; python local_cluster.py; popd`
 5. `export SSDB="127.0.0.1:6379,127.0.0.1:6380,127.0.0.1:6381"`
-5. Run the desired tests (see `make help` for more details)
+6. Run the desired tests (see `make help` for more details)
```

Comment:
This wasn't your update, but this section should now read as follows. Could you update it please?
```text
To run the tests, simply execute the following command. Omit SR_PYTHON=On or SR_FORTRAN=On if you do not wish to test those languages: 
make test SR_PYTHON=On SR_FORTRAN=On
```

Reply:
OK, done